egui_demo_lib/src/apps/demo/plot_demo.rs

```diff
@@ -626,6 +626,8 @@ impl ChartsDemo {
             }
         };
 
+        let grid = plot::uniform_grid_spacer(|_input| -> [f64; 3] { [1.0, 5.0, 30.0] });
```

Comment:
Can we come up with a better demo for the usefulness of this new feature? Dividing up days in multiples of 5 and 30 is not very common.

Reply:
Splitting time into minutes and seconds could be a good demo

Reply:
Done.